test/other/test_contrib_ports.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2024 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+#include <GLFW/glfw3.h>
```

Comment:
Is this header any different the emscirpten one?

If not perhaps we could just install `emscripten_glfw3.h` and rely on the existing header?

Reply:
I don't know if this it is the same or not. But the idea is that the port is self contained and brings its own dependencies. If it were to diverge then that would become an issue. I would rather keep it that way. Note that this `test_contrib_ports.cpp` will be replaced with another port once there is another one (when/if I work on libarchive I will switch it to this port)